### ISSUE TITLE
[expo-splash-screen] java.lang.IllegalStateException at expo.modules.splashscreen.SplashScreenController$showSplashScreen$2.run (SplashScreenController.java:12)

### ISSUE THREAD

**Issue by brunoaduarte** [expo/expo#9937]
### Summary of Issue 

```
java.lang.IllegalStateException: 
  at android.view.ViewGroup.addViewInner (ViewGroup.java:5204)
  at android.view.ViewGroup.addView (ViewGroup.java:5027)
  at android.view.ViewGroup.addView (ViewGroup.java:4964)
  at android.view.ViewGroup.addView (ViewGroup.java:4937)
  at expo.modules.splashscreen.SplashScreenController$showSplashScreen$2.run (SplashScreenController.java:12)
  at android.app.Activity.runOnUiThread (Activity.java:6970)
  at expo.modules.splashscreen.SplashScreenController.a (SplashScreenController.java:20)
  at expo.modules.splashscreen.SplashScreenController.a (SplashScreenController.java:6)
  at expo.modules.splashscreen.SplashScreenController$handleRootView$1.onChildViewRemoved (SplashScreenController.java:28)
  at android.view.ViewGroup.dispatchViewRemoved (ViewGroup.java:5105)
  at android.view.ViewGroup.removeViewInternal (ViewGroup.java:5587)
  at android.view.ViewGroup.removeViewAt (ViewGroup.java:5509)
  at com.facebook.react.uimanager.ViewGroupManager.removeViewAt (ViewGroupManager.java:3)
  at com.facebook.react.uimanager.NativeViewHierarchyManager.a (NativeViewHierarchyManager.java:149)
  at com.facebook.react.uimanager.UIViewOperationQueue$ManageChildrenOperation.execute (UIViewOperationQueue.java:16)
  at com.facebook.react.uimanager.UIViewOperationQueue$1.run (UIViewOperationQueue.java:72)
  at com.facebook.react.uimanager.UIViewOperationQueue.i (UIViewOperationQueue.java:53)
  at com.facebook.react.uimanager.UIViewOperationQueue.access$2400 (UIViewOperationQueue.java)
  at com.facebook.react.uimanager.UIViewOperationQueue$2.runGuarded (UIViewOperationQueue.java:2)
  at com.facebook.react.bridge.GuardedRunnable.run (GuardedRunnable.java)
  at android.os.Handler.handleCallback (Handler.java:914)
  at android.os.Handler.dispatchMessage (Handler.java:100)
  at android.os.Looper.loop (Looper.java:224)
  at android.app.ActivityThread.main (ActivityThread.java:7551)
  at java.lang.reflect.Method.invoke (Method.java)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:539)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:995)
```

### Environment - output of `expo diagnostics` & the platform(s) you're targeting

  Expo CLI 3.23.3 environment info:
    System:
      OS: Windows 10 10.0.18362
    Binaries:
      Node: 14.4.0 - C:\Program Files\nodejs\node.EXE
      Yarn: 1.22.4 - C:\Users\Myself\AppData\Roaming\npm\yarn.CMD     
      npm: 6.14.4 - C:\Program Files\nodejs\npm.CMD
    npmPackages:
      expo: ^38.0.8 => 38.0.8
      react: ~16.11.0 => 16.11.0
      react-dom: ~16.11.0 => 16.11.0
      react-native: ~0.62.2 => 0.62.2
      react-native-web: ~0.11.7 => 0.11.7

### Steps to Reproduce

https://forums.expo.io/t/error-on-some-android-phone/39559

**Comment by bbarthec** [comment]
Fixed by https://github.com/expo/expo/pull/9451
The recent version of `expo-splash-screen` is containing fix for this. Please upgrade to https://www.npmjs.com/package/expo-splash-screen/v/0.6.2

**Comment by jiahong96** [comment]
> Fixed by #9451
> The recent version of `expo-splash-screen` is containing fix for this. Please upgrade to https://www.npmjs.com/package/expo-splash-screen/v/0.6.2

how do I know if expo SDK 38 is compatible with the version of `expo-splash-screen` that you mentioned?

**Comment by brentvatne** [comment]
@jiahong96 - you're in the wrong thread :) expo-splash-screen wasn't even used in the managed workflow in sdk 38. i don't know what issue you are having but you could make a new issue and describe it in detail following the issue template
